src/Admin/PageAdmin.php

```diff
@@ -368,10 +368,10 @@ protected function configureTabMenu(ItemInterface $menu, string $action, ?AdminI
                 $site = $page->getSite();
 
                 if (null !== $site) {
-                    $siteRelativePath = $site->getRelativePath();
+                    $relativePath = $site->getRelativePath();
 
-                    if (null !== $siteRelativePath) {
-                        $path = $siteRelativePath.$path;
+                    if (!\in_array($relativePath, [null, '/'], true)) {
+                        $path = $relativePath.$path;
```

Comment:
This one is now tested by the first commit, there is still the other src change missing a test.